packaging/nmstate.spec

```diff
@@ -13,6 +13,7 @@ BuildArch:      noarch
 BuildRequires:  python3-devel
 BuildRequires:  python3-setuptools
 Requires:       python3-%{libname} = %{?epoch:%{epoch}:}%{version}-%{release}
+Requires:       systemd
```

Comment:
Should be `systemd-rpm-macros`

Reply:
@cathay4t In most documentations its ```systemd```. When using systemd-rpm-macros I get failed dependencies error.

Reply:
According to fedora package guide: https://docs.fedoraproject.org/en-US/packaging-guidelines/Systemd/

```
Please note that in order for the %{_unitdir} macro to exist, your package must have:

BuildRequires: systemd-rpm-macros
```

Reply:
And there is no need to specify run time requirements on systemd.

This rpm spec is for Fedora and RHEL/CentOS only.